contrib/kubespray/boot.sh

```diff
@@ -56,6 +56,8 @@ ansible-playbook -i ${HOME}/pai-deploy/cluster-cfg/hosts.yml drivers-install.yml
 
 ansible-playbook -i ${HOME}/pai-deploy/cluster-cfg/hosts.yml docker-runtime-setup.yml || exit $?
```

Comment:
same for docker runtime configuration, since master node may not have gpu, changing default runtime will break running containers

Reply:
Tested, it only run on worker node.